NEWS.rst

```diff
@@ -1,6 +1,13 @@
 History
 =======
 
+0.15 (2020-03-13)
+-----------------
+* Improve S3 chunk store: check tokens, improve timeouts and retries (#272 - #277)
+* Apply flux calibration if available (#278, #279)
+* Improve mvf_rechunk and mvf_read_benchmark scripts (#280, #281, #284)
+* Fix selection by target description (#271)
```

Comment:
Maybe add a note that Python 2 support is deprecated?

Reply:
Ah yes, missed that since I only looked at PR merge commits...